core/src/main/java/org/apache/iceberg/SnapshotScan.java

```diff
@@ -67,8 +66,7 @@ protected Long snapshotId() {
 
   protected abstract CloseableIterable<T> doPlanFiles();
 
-  @VisibleForTesting
-  ScanMetrics scanMetrics() {
+  protected ScanMetrics scanMetrics() {
```

Comment:
the original method in `BaseTableScan` was `protected` rather than package-private, and we also don't need `@VisibleForTesting` here.